cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java

```diff
@@ -124,15 +124,15 @@
                                                         "   Before Suite Function\n" +
                                                         "}\n" +
                                                        "@test:BeforeSuite\n" +
-                                                       "function beforeSuiteFunc () {\n" +
+                                                       "function beforeSuiteServiceFunc () {\n" +
                                                        "    io:println(\"I'm the before suite function!\");\n" +
```

Comment:
Shall we update the println messages as well ?

Reply:
Updated message with 009fc6d